ui-frontend/packages/catalog-ui-search/src/main/webapp/component/fields/date-helpers.tsx

```diff
@@ -122,7 +142,11 @@ export const DateHelpers = {
 
           if (
             momentWithOffset.isValid() &&
-            DateHelpers.Blueprint.commonProps.isValid(momentWithOffset.toDate())
+            DateHelpers.Blueprint.commonProps.isValid(
```

Comment:
If we move the min / max to be exported variables within this file, I think we should update the isValid method in this file to use those as the defaults.